src/QueueManager.php

```diff
@@ -150,13 +151,25 @@ public static function engine(string $name): SimpleClient
     /**
      * Push a single job onto the queue.
      *
-     * @param callable $callable  a job callable
-     * @param array $args         an array of data to set for the job
-     * @param array $options      an array of options for publishing the job
+     * @param string|array $callable Either an array of [classname, method].
```

Comment:
Should this just be an interface where you pass an instance?

This description is awkward to read. The "either" and "or" are disconnected.

Reply:
The description change looks fine. 

I'm curious why we have a class name + method name array here. Is there an issue with the class being instantiated outside of the manager and passed in? We could have a job interface with a specific method.

Reply:
We can't take an object into the QueueManager, as we can't reliably serialize objects and put them into queue messages. Using just the class/method lets the worker build a new class and call the appropriate method on it.

We do have a job interface, however it's `execute` method is not static, and the Processor that runs jobs assumes the method to call is also instance level.